client/model/files.js

```diff
@@ -91,7 +91,7 @@ class FileSystem{
                 location = "/login?next="+location.pathname;
             }
             this.obs.next(_err);
-            return Promise.reject(err);
+            return Promise.reject(_err);
```

Comment:
The error caused by this wrong variable name occurred when the server is not responding (so pretty rarely), but it produced an uncaught exception. 
With the current fix, the error is notified correctly on the website.